sql/table.go

```diff
@@ -30,7 +31,7 @@ import (
 
 func makeTableDesc(p *parser.CreateTable) (structured.TableDescriptor, error) {
 	desc := structured.TableDescriptor{}
-	desc.Name = p.Table.String()
+	desc.Name = strings.ToLower(p.Table.String())
```

Comment:
This would require us to be very very careful about when we use descriptor.Name vs tableKey vs table.Name from the parser. I don't particularly see that happening.